tensorflow/tools/tfprof/internal/tfprof_timeline.cc

```diff
@@ -327,7 +327,7 @@ void Timeline::AllocateLanes() {
       int64 start_time = tnode.second->start_micros;
       int64 end_time = tnode.second->start_micros + tnode.second->exec_micros;
       int64 l = -1;
-      for (int i = 0; i < p->lanes.size(); ++i) {
+      for (size_t i = 0; i < p->lanes.size(); ++i) {
```

Comment:
I'd go for `int64 i = 0; static_cast<size_t>(i) < p->lanes.size(); ++i`, just this one time, because it's assigned to l.